src/oscar/__init__.py

```diff
@@ -71,7 +71,6 @@ def get_version():
     'widget_tweaks',
     'haystack',
     'treebeard',
-    'sorl.thumbnail',
```

Comment:
I think this needs to move into `extras_require` so that you can install will `pip install django-oscar[sorl]`. Same for `easy_thumbnails`.

Reply:
Thanks. Done.